comms/src/peer_manager/node_identity.rs

```diff
@@ -58,6 +58,24 @@ pub struct NodeIdentity {
     public_address: RwLock<Multiaddr>,
 }
 
+impl Default for NodeIdentity {
+    fn default() -> Self {
```

Comment:
A default for NodeIdentity doesn't make sense. The rule for Defaults is being broken, specifically
`NodeIdentity::default() == NodeIdentity::default() // Not true`

Reply:
Thanks, good comment, makes sense. Fixing it now.